### PR TITLE
Feature/make a controller

### PR DESCRIPTION
**Pull request by steffendsommer**

### CODE REVIEW

File: Sources/Reset/Controllers/ResetController.swift [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+        return try U
+            .authenticate(using: payload, on: req)
+            .unwrap(or: ResetError.userNotFound)
+            .flatMap(to: Response.self) { user in
+                guard user.passwordChangeCount == payload.passwordChangeCount else {
+                    throw ResetError.tokenAlreadyUsed
+                }
+                return try config.responses.resetPasswordForm(req, user)
+        }
```
Comment (siemensikkema) [inline Sources/Reset/Controllers/ResetController.swift]:
```suggestion
            }
```

File: Sources/Reset/Tags/ResetConfigTag.swift [hunk 2]
```diff
@@ -14,8 +14,8 @@ public final class ResetConfigTag: TagRenderer {
 
 public final class ResetConfigTagData: Service {
     enum Keys: String {
-        case name = "name"
-        case baseURL = "baseURL"
+        case name
+        case baseURL
     }
 
     public var name = ""
```
Comment (siemensikkema) [inline Sources/Reset/Tags/ResetConfigTag.swift]: Not specific to this PR but why are these mutable?

**Review by siemensikkema — APPROVED** [review]
👍

**Comment by steffendsommer** [comment]
@siemensikkema ready for a re-review when you have a moment. Added all the missing stuff for now.

**Review by BrettRToomey — APPROVED** [review]
Looking really good!

**Pull request merged**